leasing/viewsets/lease.py

```diff
@@ -467,7 +467,8 @@ def get_queryset(self):  # noqa: C901
                 queryset = queryset.filter(
                     decisions__decision_maker=search_form.cleaned_data.get(
                         "decision_maker"
-                    )
+                    ),
+                    decisions__deleted__isnull=True
```

Comment:
Should this `decisions__deleted__isnull=True` check be in every case and not just when searching with decision maker? Example pic in the ticket shows that search is done with lease identifier.

Reply:
Technically yes but the issue is coming with decisions only so I just added it just here.

Also, the ticket didn't had any example pics. I added some pictures when I thought the issue was about searching with lease number but it actually is in advanced search so those pictures are not valid anymore